dashboard/app/controllers/projects_controller.rb

```diff
@@ -133,10 +133,61 @@ def public
     end
   end
 
+  def project_and_featured_project_fields
+    [
+      :storage_apps__id___id,
+      :storage_apps__storage_id___storage_id,
+      :storage_apps__value___value,
+      :storage_apps__project_type___project_type,
+      :storage_apps__published_at___published_at,
+      :featured_projects__featured_at___featured_at,
+      :featured_projects__unfeatured_at___unfeatured_at
+    ]
+  end
+
+  def combine_projects_and_featured_projects_data
+    storage_apps = "pegasus_#{CDO.rack_env}__storage_apps".to_sym
+    project_featured_project_combo_data = DASHBOARD_DB[:featured_projects].select(*project_and_featured_project_fields).
```

Comment:
please wrap very long line (80 recommended, 120 hard limit per [STYLEGUIDE.md](https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md)), and indent 2 on continuation, e.g:
```ruby
    project_featured_project_combo_data = DASHBOARD_DB[:featured_projects].
      select(*project_and_featured_project_fields).
```

Reply:
✅